client/src/components/ComboBox/ComboBox.scss

```diff
@@ -2,12 +2,21 @@
 // With the scrolling and show/hide of the field, correct spacing is critical.
 $spacing: theme('spacing.[2.5]');
 $spacing-sm: theme('spacing.5');
+$width: min(896px, 75vw);
 
 .w-combobox-container {
   @include dark-theme() {
     background-color: theme('colors.surface-tooltip');
   }
 
+  background: theme('colors.surface-page');
+  color: theme('colors.text-context');
+  border-radius: theme('borderRadius.DEFAULT');
+  box-shadow: theme('boxShadow.md');
+  outline: 10px solid transparent;
```

Comment:
Moving these to the top, so any layout-related properties are grouped together at the bottom to make it easier to read.